diff --git a/node_modules/bulma/sass/components/level.sass b/node_modules/bulma/sass/components/level.sass
index 8f73120..481976d 100644
--- a/node_modules/bulma/sass/components/level.sass
+++ b/node_modules/bulma/sass/components/level.sass
@@ -1,4 +1,4 @@
-$level-item-spacing: ($block-spacing / 2) !default
+$level-item-spacing: ($block-spacing * 0.5) !default
 
 .level
   @extend %block
diff --git a/node_modules/bulma/sass/elements/button.sass b/node_modules/bulma/sass/elements/button.sass
index 4bdf253..a4748a3 100644
--- a/node_modules/bulma/sass/elements/button.sass
+++ b/node_modules/bulma/sass/elements/button.sass
@@ -71,14 +71,14 @@ $button-static-border-color: $border !default
       height: 1.5em
       width: 1.5em
     &:first-child:not(:last-child)
-      +ltr-property("margin", calc(#{-1 / 2 * $button-padding-horizontal} - #{$button-border-width}), false)
-      +ltr-property("margin", $button-padding-horizontal / 4)
+      +ltr-property("margin", calc(#{-1 * 0.5 * $button-padding-horizontal} - #{$button-border-width}), false)
+      +ltr-property("margin", $button-padding-horizontal * 0.25)
     &:last-child:not(:first-child)
-      +ltr-property("margin", $button-padding-horizontal / 4, false)
-      +ltr-property("margin", calc(#{-1 / 2 * $button-padding-horizontal} - #{$button-border-width}))
+      +ltr-property("margin", $button-padding-horizontal * 0.25, false)
+      +ltr-property("margin", calc(#{-1 * 0.5 * $button-padding-horizontal} - #{$button-border-width}))
     &:first-child:last-child
-      margin-left: calc(#{-1 / 2 * $button-padding-horizontal} - #{$button-border-width})
-      margin-right: calc(#{-1 / 2 * $button-padding-horizontal} - #{$button-border-width})
+      margin-left: calc(#{-1 * 0.5 * $button-padding-horizontal} - #{$button-border-width})
+      margin-right: calc(#{-1 * 0.5 * $button-padding-horizontal} - #{$button-border-width})
   // States
   &:hover,
   &.is-hovered
diff --git a/node_modules/bulma/sass/grid/columns.sass b/node_modules/bulma/sass/grid/columns.sass
index 34a8353..8463d3a 100644
--- a/node_modules/bulma/sass/grid/columns.sass
+++ b/node_modules/bulma/sass/grid/columns.sass
@@ -1,3 +1,5 @@
+@use "sass:math"
+
 $column-gap: 0.75rem !default
 
 .column
@@ -59,9 +61,9 @@ $column-gap: 0.75rem !default
   @for $i from 0 through 12
     .columns.is-mobile > &.is-#{$i}
       flex: none
-      width: percentage($i / 12)
+      width: percentage(math.div($i, 12))
     .columns.is-mobile > &.is-offset-#{$i}
-      margin-left: percentage($i / 12)
+      margin-left: percentage(math.div($i, 12))
   +mobile
     &.is-narrow-mobile
       flex: none
@@ -116,9 +118,9 @@ $column-gap: 0.75rem !default
     @for $i from 0 through 12
       &.is-#{$i}-mobile
         flex: none
-        width: percentage($i / 12)
+        width: percentage(math.div($i, 12))
       &.is-offset-#{$i}-mobile
-        margin-left: percentage($i / 12)
+        margin-left: percentage(math.div($i, 12))
   +tablet
     &.is-narrow,
     &.is-narrow-tablet
@@ -194,10 +196,10 @@ $column-gap: 0.75rem !default
       &.is-#{$i},
       &.is-#{$i}-tablet
         flex: none
-        width: percentage($i / 12)
+        width: percentage(math.div($i, 12))
       &.is-offset-#{$i},
       &.is-offset-#{$i}-tablet
-        margin-left: percentage($i / 12)
+        margin-left: percentage(math.div($i, 12))
   +touch
     &.is-narrow-touch
       flex: none
@@ -252,9 +254,9 @@ $column-gap: 0.75rem !default
     @for $i from 0 through 12
       &.is-#{$i}-touch
         flex: none
-        width: percentage($i / 12)
+        width: percentage(math.div($i, 12))
       &.is-offset-#{$i}-touch
-        margin-left: percentage($i / 12)
+        margin-left: percentage(math.div($i, 12))
   +desktop
     &.is-narrow-desktop
       flex: none
@@ -309,9 +311,9 @@ $column-gap: 0.75rem !default
     @for $i from 0 through 12
       &.is-#{$i}-desktop
         flex: none
-        width: percentage($i / 12)
+        width: percentage(math.div($i, 12))
       &.is-offset-#{$i}-desktop
-        margin-left: percentage($i / 12)
+        margin-left: percentage(math.div($i, 12))
   +widescreen
     &.is-narrow-widescreen
       flex: none
@@ -366,9 +368,9 @@ $column-gap: 0.75rem !default
     @for $i from 0 through 12
       &.is-#{$i}-widescreen
         flex: none
-        width: percentage($i / 12)
+        width: percentage(math.div($i, 12))
       &.is-offset-#{$i}-widescreen
-        margin-left: percentage($i / 12)
+        margin-left: percentage(math.div($i, 12))
   +fullhd
     &.is-narrow-fullhd
       flex: none
@@ -423,9 +425,9 @@ $column-gap: 0.75rem !default
     @for $i from 0 through 12
       &.is-#{$i}-fullhd
         flex: none
-        width: percentage($i / 12)
+        width: percentage(math.div($i, 12))
       &.is-offset-#{$i}-fullhd
-        margin-left: percentage($i / 12)
+        margin-left: percentage(math.div($i, 12))
 
 .columns
   margin-left: (-$column-gap)
diff --git a/node_modules/bulma/sass/grid/tiles.sass b/node_modules/bulma/sass/grid/tiles.sass
index 15648c2..13bf15e 100644
--- a/node_modules/bulma/sass/grid/tiles.sass
+++ b/node_modules/bulma/sass/grid/tiles.sass
@@ -1,3 +1,5 @@
+@use "sass:math"
+
 $tile-spacing: 0.75rem !default
 
 .tile
@@ -31,4 +33,4 @@ $tile-spacing: 0.75rem !default
     @for $i from 1 through 12
       &.is-#{$i}
         flex: none
-        width: ($i / 12) * 100%
+        width: math.div($i, 12) * 100%
diff --git a/node_modules/bulma/sass/utilities/functions.sass b/node_modules/bulma/sass/utilities/functions.sass
index 270121f..bd5d00e 100644
--- a/node_modules/bulma/sass/utilities/functions.sass
+++ b/node_modules/bulma/sass/utilities/functions.sass
@@ -1,3 +1,5 @@
+@use "sass:math"
+
 @function mergeColorMaps($bulma-colors, $custom-colors)
   // We return at least Bulma's hard-coded colors
   $merged-colors: $bulma-colors
@@ -58,7 +60,7 @@
       $value: $value * $number
   @else if $exp < 0
     @for $i from 1 through -$exp
-      $value: $value / $number
+      $value: math.div($value, $number)
   @return $value
 
 @function colorLuminance($color)
@@ -67,11 +69,11 @@
   $color-rgb: ('red': red($color),'green': green($color),'blue': blue($color))
   @each $name, $value in $color-rgb
     $adjusted: 0
-    $value: $value / 255
+    $value: math.div($value, 255)
     @if $value < 0.03928
-      $value: $value / 12.92
+      $value: math.div($value, 12.92)
     @else
-      $value: ($value + .055) / 1.055
+      $value: math.div($value + .055, 1.055)
       $value: powerNumber($value, 2)
     $color-rgb: map-merge($color-rgb, ($name: $value))
   @return (map-get($color-rgb, 'red') * .2126) + (map-get($color-rgb, 'green') * .7152) + (map-get($color-rgb, 'blue') * .0722)
